src/components/LabelValueMetadata.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Renders label/value pair metadata
+ * (e.g. what comes from manifesto) in a dl
+ * @prop {object} labelValuePair
+ */
+class LabelValueMetadata extends Component {
+  /**
+   * render
+   * @return {String} - HTML markup for the component
+   */
+  render() {
+    const { labelValuePairs } = this.props;
+
+    if (labelValuePairs.length === 0) {
+      return (<></>);
+    }
+
+    return (
+      <dl>
+        {labelValuePairs.reduce((acc, labelValuePair, i) => acc.concat([
+          <dt key={`label-${JSON.stringify(labelValuePair.label[0].value)}`}>
+            {labelValuePair.label[0].value}
+          </dt>,
+          <dd key={`value-${JSON.stringify(labelValuePair.value[0].value)}`}>
+            {labelValuePair.value[0].value}
+          </dd>,
```

Comment:
I think the component should get label-value-pairs that are already resolved like `[{ label: "foo", value: 1 }, { label: "bar", value: 2 }]`. In the above code the component depends on the peculiarities of the iiif manifest and has to do data transformation. To change this in this way would probably imply to write more specific selectors and pass them to <WindowSideBarInfoPanel/>.

Reply:
Makes sense.  I'll look into pushing this logic elsewhere.

Reply:
Am I right that part of the peculiarity of the data structure we're getting from manifesto is something to do with multilingual data (and that flattening it may lose some of the connections expressed in IIIF)?

Reply:
@cbeer correct.  My plan will be to abstract this logic somewhere (most likely a selector) that can have knowledge about which locale is set (once we have that functionality).  Then in the container, where we're mapping state to props, we can use the locale from state when fetching the metadata, and pass the correct thing to the presentation component. (at least that's where my thinking is going so far)